### ISSUE TITLE
Will pgjsonb be upgraded?

### ISSUE THREAD

**Issue by zhaodefangde** [vzornic/pg-jsonb#9]
pgjsonb strongly depends on hibernate 5.6, but in hibernate 6.0, many classes no longer exist, do you have any plans to upgrade pgjsonb according to hibernate 6.0?

**Comment by vzornic** [comment]
No, this lib is deprecated at this point due to much better support in https://vladmihalcea.com/ libs. If you still want to use this feel free to modify and open PR
